docs/hosting-packages/Overview.md

```diff
@@ -26,11 +26,11 @@ There are also several other NuGet hosting products such as [Azure Artifacts](ht
 - [Cloudsmith](https://cloudsmith.io/l/nuget-feed/), a fully managed package management SaaS
 - [GitHub package registry](https://help.github.com/articles/configuring-nuget-for-use-with-github-package-registry)
 - [LiGet](https://github.com/ai-traders/liget), an open-source implementation of NuGet V2 server that runs on kestrel in docker
-- [MyGet](http://myget.org)
-- [Nexus](http://www.sonatype.org/nexus/) from Sonatype.
-- [NuGet Server (Open Source)](http://nuget-server.net), an open-source implementation similar to Inedo's NuGet Server
+- [MyGet](https://myget.org)
+- [Nexus](https://www.sonatype.org/nexus/) from Sonatype.
+- [NuGet Server (Open Source)](https://github.com/svenkle/nuget-server), an open-source implementation similar to Inedo's NuGet Server
```

Comment:
**Note:** I added the github link directly because nuget-server.net redirects to it. so I put it directly.